docs/source/appendix.rst

```diff
@@ -1,8 +1,62 @@
-Zipline API
------------
+API Reference
+-------------
 
-For each of these api functions, the ``self`` argument is implicitly the current
-:class:`~zipline.algorithm.TradingAlgorithm`
+Algorithm API
+~~~~~~~~~~~~~
+
+The following methods are available for use in the ``initialize``,
+``handle_data`` and ``before_trading_start`` API functions.
```

Comment:
There are raging debates over whether or not it's correct to put a comma here: http://www.oxforddictionaries.com/us/words/what-is-the-oxford-comma.